### PR TITLE
Feature | i18n

### PR DESCRIPTION
**Pull request by Pakholchuk-Oleh**

### CODE REVIEW

**Review by mmrshk — CHANGES_REQUESTED** [review]
5 thousands files???

firstly, divide, why you did two features in one pr?
secondly, why do you need webpack?

**Comment by mmrshk** [comment]
And also vendor files must be in .gitignore

**Review by Andriy1328 — APPROVED** [review]
LGTM

File: config/application.rb [hunk 1]
```diff
@@ -15,5 +15,7 @@ class Application < Rails::Application
     # Application configuration can go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded after loading
     # the framework and any gems in your application.
+    config.i18n.available_locales = [:en, :ua]
+    config.i18n.default_locale = :en
```
Comment (mmrshk) [inline config/application.rb:19]: I think for this site default locale will be `ua`.

**Review by mmrshk — APPROVED** [review]
Fix comments and merge.

**Comment by mmrshk** [comment]
> Fix comments and merge.

Also please in the future add description to the task. 
1. Link to the task
2. What have you done in this task?

**Pull request merged**